### ISSUE TITLE
Find baseline code from internet

### ISSUE THREAD

**Issue by aimanmuhamad** [Drithh/e-commerce-website#41]

**Comment by aimanmuhamad** [comment]
Kaggle Link :

**Comment by DaffaKR** [comment]
https://www.kaggle.com/code/arezoodahesh/fashion-mnist-cnn-model

https://www.kaggle.com/code/bhatnagardaksh/data-augmentation-lr-tracking-decay

https://www.kaggle.com/code/andreasstavrou/the-almost-perfect-experimentation-ensemble-cnns

**Comment by galihindra22** [comment]
https://www.kaggle.com/code/fuzzywizard/fashion-mnist-cnn-keras-accuracy-93
https://www.kaggle.com/code/pankajj/fashion-mnist-with-pytorch-93-accuracy
https://www.kaggle.com/code/littleraj30/in-depth-keras-v-s-pytorch-approach-comparison
https://www.kaggle.com/code/nohrud/fashion-mnist-with-pytorch
https://www.kaggle.com/code/tinydman/fashion-mnist-with-pytorch
